comps/router/deployment/docker_compose/config.yaml

```diff
@@ -0,0 +1,17 @@
+# Global config for your router microservice
+# TODO: Update endpoint naming
+model_map:
+  weak:
+    endpoint: "http://some-inference-service:8000/weak"
```

Comment:
possible to make it configurable via env varible? by the way, default value might be http://opea_router:8000/weak using docker_name